setup/base/base.go

```diff
@@ -100,6 +100,7 @@ const (
 	DisableMetrics BaseDendriteOptions = iota
 	UseHTTPAPIs
 	PolylithMode
+	CreateAccountMode
```

Comment:
Rather than changing the shape of `base`, can we instead just overwrite the registration disabled flag in the loaded `cfg` in `cmd/create-account/main.go` to `true` before handing it to `NewBaseDendrite`?